Frontend/src/app/components/bottom-bar/bottom-bar.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { BottomBarComponent } from './bottom-bar.component';
+
+describe('BottomBarComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7378189)
```suggestion
describe("BottomBarComponent", () => {
```